.github/workflows/bot.yml

```diff
@@ -285,15 +279,13 @@ jobs:
           SCALA_PROFILE: ${{ matrix.scalaProfile }}
           SPARK_PROFILE: ${{ matrix.sparkProfile }}
           SPARK_MODULES: ${{ matrix.sparkModules }}
-        if: ${{ !endsWith(env.SPARK_PROFILE, '3.2') }} # skip test spark 3.2 as it's covered by Azure CI
```

Comment:
why are we removing this? are we going to have duplicate tests for spark3.2 ?

Reply:
These tests only run on Spark 3.3 and above so there is no point having this condition (which is always `true` before this PR).